src/runtime/schema/settings-mapper.ts

```diff
@@ -0,0 +1,79 @@
+/**
```

Comment:
Extracting this felt good. So much distraction from the core concern of user-facing framework settings here.